### DISCUSSION TITLE
Adding theme CSS for specific component

### DISCUSSION THREAD

**Discussion by k-mahajan** [primefaces/community#1334] [PrimeVue] [answered]
Can I add CSS for specific component only.
In my case, I want to add Theme CSS for Autocomplete component only. But while importing CSS in main.js, Theme CSS is applied to whole project.

**Comment by tugcekucukoglu** [comment, marked as answer]
Every component has `unstyled` property and if you choose to use a theme and don't want to use it for a specific component. Therefore `unstyled` property can be set as false.
